easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2021a.eb

```
@@ -0,0 +1,44 @@
+# Author: Denis Krišťák (INUITS)
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Update: Erica Bianco <erica.bianco@hpcnow.com>
+# License::     GPL-v3.0
+easyblock = "PythonBundle"
+
+name = 'umap-learn'
+version = '0.5.3'
+
+homepage = 'https://umap-learn.readthedocs.io/en/latest/'
+description = """Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique 
+                 that can be used for visualisation similarly to t-SNE, but also for general non-linear 
+                 dimension reduction."""
```

the whitespace will be preserved in the generated module file, so I would suggest something like this
```suggestion
description = """
Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique 
that can be used for visualisation similarly to t-SNE, but also for general non-linear 
dimension reduction.
"""
```